promgen/templates/promgen/help_menu.inc.html

```diff
@@ -1,7 +1,7 @@
 <li class="dropdown">
     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Help <span class="caret"></span></a>
     <ul class="dropdown-menu">
-        <li><a rel="noopener noreferrer" href="https://promgen.readthedocs.io/">Documentation</a></li>
+        <li><a rel="noopener noreferrer" href="https://https://line.github.io/promgen/">Documentation</a></li>
```

Comment:
There is a duplication of `https://` in that URL.

Reply:
Fixed and force pushed with a fixed commit messages.